training/distributed_training/Dockerfile

```diff
@@ -0,0 +1,101 @@
+FROM nvidia/cuda:9.0-devel-ubuntu16.04
+
+# TensorFlow version is tightly coupled to CUDA and cuDNN so it should be selected carefully
+ENV TENSORFLOW_VERSION=1.10.0
+ENV CUDNN_VERSION=7.0.5.15-1+cuda9.0
+ENV NCCL_VERSION=2.2.13-1+cuda9.0
+
+
+
+RUN apt-get update && apt-get install -y --no-install-recommends --allow-downgrades --allow-change-held-packages \
+      build-essential \
+      cmake \
+      git \
+      curl \
+      vim \
+      wget \
+      ca-certificates \
+      libcudnn7=${CUDNN_VERSION} \
+      libnccl2=${NCCL_VERSION} \
+      libnccl-dev=${NCCL_VERSION} \
+      libjpeg-dev \
+      libpng-dev  \
+      openssl \
+      libssl-dev \
+      unzip \
+      net-tools
+
+# Install Python3
+RUN wget https://www.python.org/ftp/python/3.6.6/Python-3.6.6.tgz
```

Comment:
Does this example require Python 3.6? If not, Python 3.5 is much easier to install: `apt-get install python3.5`

Reply:
I dont think 3.6 is requirement from example but having 3.6 is still better in case if we add any other example tied to higher version of python.